debezium-connector-ibmi-3.0.yaml

```diff
@@ -0,0 +1,81 @@
+package:
+  name: debezium-connector-ibmi-3.0
+  version: 3.0.2
+  epoch: 0
+  description: Debezium Connector for IBM i (AS/400)
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provides:
+      - debezium-connector-ibmi=${{package.full-version}}
+      - debezium-3.0-connector-ibmi=${{package.full-version}}
```

Comment:
Why do we need this?

Reply:
Actually to follow `package-xyz` format and make the version streaming easier in the image, otherwise we would have version stream for each package, IINM.